src/js/actions.js

```diff
@@ -146,9 +146,9 @@ export const deleteSubMenu = (appID, menuID) => {
 export const subscribeButton = (appID, buttonName, isSubscribed) => {
     return {
         type: Actions.SUBSCRIBE_BUTTON,
-        appID: appID,
-        buttonName: buttonName,
-        isSubscribed: isSubscribed
+        appID,
+        buttonName,
+        isSubscribed
```

Comment:
```suggestion
        appID: appID,
        buttonName: buttonName,
        isSubscribed: isSubscribed
```
reverting this, otherwise it is inconsistent with the rest of the file

Reply:
@jacobkeeler please find fix [here](https://github.com/LuxoftSDL/generic_hmi/pull/26/commits/c1a08c356f3f507001a80145f48d31b33c44b097).